store/database.go

```diff
@@ -118,11 +290,26 @@ func (s *DatabaseService) FindDatabaseList(ctx context.Context, find *api.Databa
 	return list, nil
 }
 
-// FindDatabase retrieves a single database based on find.
+func (s *Store) getDatabaseRawWrapped(ctx context.Context, find *api.DatabaseFind) (*api.Database, error) {
```

Comment:
Is there any reason to keep both getDatabase and findDatabase? Is there any way to simplify or reduce the number of find methods?

Reply:
`FindDatabase` returns a list and `GetDatabase` returns an instance.
Removed redundant variants of `GetDatabase` in the following commit.